PSI-MOD.obo

```diff
@@ -12582,29 +12482,31 @@ xref: TermSpec: "none"
 xref: Unimod: "Unimod:154"
 xref: GNOme: "GNO:G64686LL"
 is_a: MOD:00725 ! complex glycosylation
+is_a: MOD:00160 ! N4-glycosyl-L-asparagine
 
 [Term]
 id: MOD:00523
-name: Hex1HexNAc2dHex1Pent1
+name: Hex1HexNAc2dHex1Pent1 N4-glycosylated asparagine
 def: "modification from Unimod N-linked glycosylation" [RESID:AA0151#var, Unimod:155]
 synonym: "Hex(1)HexNAc(2)dHex(1)Pent(1)" RELATED PSI-MS-label []
 synonym: "Hex1HexNAc2dHex1Pent1" RELATED Unimod-description []
 xref: DiffAvg: "846.79"
 xref: DiffFormula: "C 33 H 54 N 2 O 23"
 xref: DiffMono: "846.311736"
 xref: Formula: "C 35 H 57 N 3 O 26"
-xref: MassAvg: "935.84"
-xref: MassMono: "935.323029"
+xref: MassAvg: "960.89"

```

Comment:
naively I'd think that original mass is right because 846 + 114 (Asn) ~= 960. But I'm betting there's a good reason you changed it.